resources/images/icons/icon-multipart.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+   width="30px" height="30px" viewBox="0 0 30 30" enable-background="new 0 0 30 30" xml:space="preserve">
+<g>
+  <path d="M15.9495652,27.3078261 C22.0565217,26.8565217 26.9217391,21.9913043 27.3730435,15.8843478 L15.9495652,15.8843478 L15.9495652,27.3078261 L15.9495652,27.3078261 Z" id="Shape" fill="#D1D1D1"></path>
+  <path d="M14.1052174,2.61652174 C7.99565217,3.06782609 3.13304348,7.93434783 2.68173913,14.04 L14.1052174,14.04 L14.1052174,2.61652174 L14.1052174,2.61652174 Z" id="Shape" fill="#D1D1D1"></path>
+  <path d="M2.68304348,15.8843478 C3.13434783,21.9913043 7.99826087,26.8565217 14.1065217,27.3078261 L14.1065217,15.8843478 L2.68304348,15.8843478 L2.68304348,15.8843478 Z" id="Shape" fill="#D1D1D1"></path>
+  <path d="M15.9495652,14.0413043 L27.39,14.0413043 C26.943913,7.92391304 22.0669565,3.04826087 15.9495652,2.60478261 L15.9495652,14.0413043 L15.9495652,14.0413043 Z" id="Shape" fill="#F9BAA2"></path>
+  <path d="M29.373913,15.0117391 C29.373913,14.9947826 29.3765217,14.9791304 29.3765217,14.9608696 C29.3765217,14.9426087 29.373913,14.9282609 29.373913,14.91 L29.373913,14.04 L29.3308696,14.04 C28.8717391,6.85304348 23.1352174,1.11652174 15.9495652,0.66 L15.9495652,0.613043478 L14.1065217,0.613043478 L14.1065217,0.66 C6.91956522,1.11782609 1.18565217,6.85434783 0.726521739,14.04 L0.683478261,14.04 L0.683478261,14.91 C0.683478261,14.9269565 0.680869565,14.9426087 0.680869565,14.9608696 C0.680869565,14.9791304 0.683478261,14.9934783 0.683478261,15.0117391 L0.683478261,15.8830435 L0.727826087,15.8830435 C1.18565217,23.07 6.91956522,28.803913 14.1065217,29.2630435 L14.1065217,29.3034783 L14.9152174,29.3034783 C14.9530435,29.3034783 14.9908696,29.3086957 15.0286957,29.3086957 C15.0665217,29.3086957 15.1043478,29.3034783 15.1421739,29.3034783 L15.9495652,29.3034783 L15.9495652,29.2630435 C23.1352174,28.803913 28.8717391,23.07 29.3295652,15.8830435 L29.373913,15.8830435 L29.373913,15.0117391 L29.373913,15.0117391 Z M27.39,14.0413043 L15.9495652,14.0413043 L15.9495652,2.60478261 C22.0669565,3.04695652 26.943913,7.9226087 27.39,14.0413043 L27.39,14.0413043 Z M14.1052174,2.61652174 L14.1052174,14.04 L2.68173913,14.04 C3.13304348,7.93434783 7.99695652,3.06782609 14.1052174,2.61652174 L14.1052174,2.61652174 Z M2.68304348,15.8843478 L14.1065217,15.8843478 L14.1065217,27.3078261 C7.99826087,26.8565217 3.13304348,21.99 2.68304348,15.8843478 L2.68304348,15.8843478 Z M15.9495652,27.3078261 L15.9495652,15.8843478 L27.3730435,15.8843478 C26.9217391,21.99 22.0552174,26.8565217 15.9495652,27.3078261 L15.9495652,27.3078261 Z" id="Shape" fill="#A1A1A1"></path>
+</g>
+</svg>
```

Comment:
Hey, have you tested building for production using this in tutor?  I'm curious if they get build properly, since Tutor might not know about the path to the svg icon.

In #70 I made the SVG into [lightweight react-components](https://github.com/openstax/react-components/blob/70b2dd5d60886ffc5cfe932a4e379f6b08b25089/src/components/exercise/preview/interactive-icon.cjsx) for that reason.